marmoset/dhcp/dhcp_config.py

```diff
@@ -0,0 +1,68 @@
+from .isc_dhcp_ldap_config import ISCDhcpLdapConfig
+from marmoset import validation
+import socket
+
+
+class DhcpConfig:
+    def __init__(self, mac, ip_address, gateway=None, networkmask=None):
+        self.additional_statements = {}
```

Comment:
this adds all other supplied params to the attribute additional_statements?

please have a look at https://github.com/virtapi/marmoset/issues/7#issuecomment-175262281